dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/utils/FileResourceUtils.java

```diff
@@ -282,7 +284,7 @@ private static void validateFileExtension(String fileName, List<String> validExt
     }
   }
 
-  private static void validateFileSize(@Nonnull MultipartFile file, long maxFileSizeInBytes) {
+  public static void validateFileSize(@Nonnull MultipartFile file, long maxFileSizeInBytes) {
```

Comment:
we could probably remove the file size param & just use the constant every time, unless other callers pass different sizes for different reasons?

Reply:
can see it's called in another PR with a different size. Can ignore that comment 👆